_includes/tutorials/transforming/ksql/code/tutorial-steps/dev/outputs/transient-query/output-0.log

```diff
@@ -1,6 +1,9 @@
-294 | Die Hard | 1988 | action
-354 | Tree of Life | 2011 | drama
-782 | A Walk in the Clouds | 1995 | romance
-128 | The Big Lebowski | 1998 | comedy
++--------------------------------------------+--------------------------------------------+--------------------------------------------+--------------------------------------------+
```

Comment:
Looks like the corresponding expected output needs to change in
`tutorial-steps/dev/expected-transient.log`

+------------+------------+------------+------------+
|ID          |TITLE       |YEAR        |GENRE       |
+------------+------------+------------+------------+
|294         |Die Hard    |1988        |action      |
|354         |Tree of Life|2011        |drama       |
|782         |A Walk in th|1995        |romance     |
|            |e Clouds    |            |            |
|128         |The Big Lebo|1998        |comedy      |
|            |wski        |            |            |

Reply:
done, thanks for the tips on this